community/neatvi/build

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh -e
+
+sed -i -e '/CFLAGS = /c\' -e 'CFLAGS += -Wall' Makefile
```

Comment:
Any reason for this line?

Reply:
It's overwriting the global CFLAGS otherwise, no?

Reply:
You are correct, though it's also adding `-Wall`?

Could this not be: `sed -i 's/CFLAGS =/CFLAGS +=/' Makefile`?

Reply:
No, the original Makefile also sets `-O2`. Appending that to `CFLAGS` overwrites the users optimization level. My first idea was to simply remove that entire line but thought maybe it's better to retain warnings. Two substitutions, one to change `=` to `+=`, and one to remove `-O2` would have worked as well but I found this prettier.